packages/website/src/components/lib/markdown.ts

```diff
@@ -14,41 +14,46 @@ export function createSummary(
   return code;
 }
 
-function createSummaryJson(
-  obj: ConfigModel['tsconfig'] | ConfigModel['eslintrc'],
-  field: string,
-  title: string,
-): string {
-  if (obj && Object.keys(obj).length > 0) {
+function createSummaryJson(obj: string, field: string, title: string): string {
+  if (obj && obj.length > 0) {
     return createSummary(obj, title, 'json', 10);
   }
   return '';
 }
 
-export function genVersions(state: ConfigModel): string {
+/**
+ * Generate a table of versions
+ */
+export function genVersions(tsVersion: string): string {
```

Comment:
[Nit] Maybe a informative function name would remove the need for a comment?

```suggestion
export function generateVersionsTable(tsVersion: string): string {
```